_posts/en/newsletters/2024-05-17-newsletter.md

```diff
@@ -0,0 +1,185 @@
+  describes several problems, suggests solutions for many of them,
+  and solicits feedback on his suggestions as well as proposals for
+  solutions to the remaining problems.  For previous discussion about
+  updating BIP2, see [Newsletter #297][news297 bip2]. <!-- TODO
+  (optional): harding to add topic page for something like
+  "Specification management" to track BIP and BINANA meta discussion -->
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.18.0-beta.rc2][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Core Lightning #7190][] adds an additional offset `chainlag` into the CLTV
+  value calculation to correctly target the current block height instead of the
+  sync height. Payments are now safe and allowed during the blockchain sync
+  process.
+
+- [LDK #2973][] implements support for `OnionMessenger` to intercept messages on
+  behalf of offline peers. It generates events on message interception and on
+  the peer's return to online status for forwarding. Users should maintain an
+  allow list to only store messages for relevant peers. This is a stepping stone
+  to supporting offline payments through `held_htlc_available` [BOLTs
+  #989][], which would allow an LSP (Lightning Service Provider) to hold onto
+  an [HTLC][topic htlc] until a user comes back online.
+
+- [LDK #2907][] extends `OnionMessage` handling to accept an optional
+  `Responder` input and return an object `ResponseInstructions` that contains
+  the `response` message and the `reply_path`. This change enables asynchronous
+  onion messaging responses and opens the door to more complex response
+  mechanisms.
+
+- [BDK #1403][] updates the `bdk_electrum` crate to make use of new
+  sync/full-scan structures introduced in [BDK #1413][], queryable `CheckPoint`
+  linked list [BDK #1369][], and cheaply-clonable transactions in `Arc`
+  pointers [BDK #1373][]. This change improves performance on wallet sync and
+  transaction data scan. It is also now an option to fetch `TxOut`s to allow for
+  fee calculation on transactions received from an external wallet.
+
+- [BIPs #1458][] assigns [BIP352][] to the [Silent Payments][topic silent
+  payments] proposal by Josie Baker and Ruben Somsen for a protocol for reusable
```

Comment:
I think we mostly avoid names in this section 
```suggestion
  payments] proposal for a protocol for reusable
```

Reply:
As background for @Gustavojfe: we avoid names in the Merges section because getting something merged is a collaborative effort between both the authors and reviewers.  For code projects, a merge of code also usually indicates that it represents the rough consensus opinion of the project's maintainers (this obviously doesn't apply to BIPs).  It would be a lot of effort to list all of the reviewers[1], so we also don't list authors in order to be (somewhat) fair.

We do list names in the News section because we want to attribute ideas and opinions to individuals, since they may be positions that don't have widespread support.

[1] Especially since sometimes reviewers are spread across multiple places, as is often the case for BIPs, where review happens on the mailing lists, on the BIP repos, and often through personal correspondence and small-group discussion.